### PR TITLE
Use EKF core as instance number when logging EKF timings

### PR DESCRIPTION
**Pull request by peterbarker**
Also adds units to this message.

May make things slightly faster as we won't be creating as many dynamic log messages this way.

### CODE REVIEW

File: libraries/AP_NavEKF/AP_Nav_Common.cpp [hunk 1]
```diff
@@ -6,13 +6,16 @@
...
   write an EKF timing message
  */
-void Log_EKF_Timing(const char *name, uint64_t time_us, const struct ekf_timing &timing)
+void Log_EKF_Timing(const char * name, const uint8_t core, uint64_t time_us, const struct ekf_timing &timing)
 {
     AP::logger().Write(
         name,
-        "TimeUS,Cnt,IMUMin,IMUMax,EKFMin,EKFMax,AngMin,AngMax,VMin,VMax",
-        "QIffffffff",
+        "TimeUS,C,Cnt,IMUMin,IMUMax,EKFMin,EKFMax,AngMin,AngMax,VMin,VMax",
```
Comment (rmackay9) [inline libraries/AP_NavEKF/AP_Nav_Common.cpp]: I guess we couldn't fit "Core" or "Instance" in the title instead of just "C"?

**Comment by rmackay9** [comment]
Looks pretty good to me.  I'm actually surprised this makes things faster.. I know that if we do a block write from a structure that's faster.. I'm sure you're right of course, nobody would know better.

**Comment by WickedShell** [comment]
@rmackay9 Blocks would be faster still, this mostly just avoids the fact that you have to do a linear search to find the log ID for the string name when you log this method, so because this removes some of the extra string entries to search through you find the result a bit faster and thus save time. It's a pretty small amount though :)

File: libraries/AP_NavEKF/AP_Nav_Common.cpp (hunk 1 above)
Comment (WickedShell) [reply libraries/AP_NavEKF/AP_Nav_Common.cpp]: There's a 64 char limit, which this is exactly at with `C`. So if we can remove 3 letters elsewhere then we can have `Core`....

**Pull request merged**